DS13/code/modules/EMH/EMH.dm

```diff
@@ -0,0 +1,291 @@
+GLOBAL_LIST_INIT(EMH_blacklist, list())
+
+//EMITTER
+
+/obj/machinery/emh_emitter
+	name = "Emergency medical hologram emitter"
+	desc = "Simply click this device and an emergency medical hologram will be summoned. Swipe it with a captain level ID to terminate the current EMH if it's misbehaving."
+	icon = 'DS13/icons/obj/decor/wall_decor.dmi'
+	icon_state = "emh-off"
+	anchored = TRUE
+	density = FALSE
+	pixel_y = 32 //Put on the tile below a wall etc etc modular fun time mapping mayhem
+	var/range = 10 //10 tiles? Pretty damn generous.
+	var/mob/living/carbon/human/species/holographic/emh
+	req_access = list(ACCESS_CAPTAIN)
```

Comment:
Perhaps CMO instead we dont want a EMH to do captain stuff if its confined to medical.